### ISSUE TITLE
"Object reference not set to an instance of an object"

### ISSUE THREAD

**Issue by goosedoeshonk** [mikeoverbay/TankExporter#110]
idk what to do, is this just a bug in the code? happens while loading node 01

**Comment by damascusm** [comment]
same issue

**Comment by Cerberus50** [comment]
There are 3 "gui-parts" 1. 2 and 3 in the "res/packages" of the game. There were only 2 before. This is why T.E crashes

**Comment by Cerberus50** [comment]
I made a backup copy of the gui-part2, I opened the gui-part2 and gui-part3 with winrar and I emptied the part-3 into the part-2 then I renamed the gui-part3 in gui-part3nul.

And TE was resurrected!  But WOT di'nt WORK NOW

**warning** made a back up of the 3 gui-part before processed .

 COPY/PASTE 3 back up gui-part to RES/PACKAGE after CLOSE T.E. And WoT works now

**Comment by 33dead33king33** [comment]
isnt there a easier way to fix it?

**Comment by Cerberus50** [comment]
Yes fix T.E but I don't know how to do it

**Comment by QuietShow** [comment]
**WARNING1:** BEFORE DOING ANYTHING, DO A BACKUP OF gui-part2 AND gui-part3 INSIDE res/packages IN WOT FILES.
nazref, or anyone needing help, it's not hard to make what he said, but make sure to make a backup of gui-part3 and gui-part2, because if you are going to empty the gui-part3 into gui-part2, you are going to modify the gui-part2.

I'll call the 1st gui-part2 and gui-part3 MAIN.

An easier way to do is to extract gui-part3 to [your choice, as long as you know where you are extracting it] and then open gui-part2 with winrar and copy the files from gui-part3 and paste them inside gui-part2. After that you may change the name of the gui-part3, like Cerberus did, to gui-part3nul, or just remove the gui-part3 --> I removed.

PS.: If you can't extract the .pkg file (the MAIN file), which is normal, open winrar and extract the folder that pops up when you open MAIN gui-part2 not the files inside that folder.

**WARNING2:** AFTER THESE STEPS, DO A BACKUP OF THE gui-part2 THAT YOU'VE CHANGED, BUT **DO NOT** REPLACE THE MAIN gui-part2 DONE IN [WARNING1], so you don't need to repeat everytime this process.

After that WoT will no longer work. What to do? Copy and Paste the MAIN files into WoT res/packages file once again. IF YOU CHANGED THE gui-part3 TO gui-part3nul **REMOVE** THE gui-part3nul and add the MAIN files.

AGAIN, BEFORE EDITING SOMETHING:
- ALWAYS make a backup
- if you don't know what you're doing tell me, because I eventually can make a tutorial until mikeoverbay doesn't fix this.

**Comment by nazref** [comment]
I have managed to do it and it works. Just copy all gui3 to gui2(copy) and rename gui2(copy) to "gui 2 (or whatever the default is)"  and dont need to remove gui3 cuz T.E dont check the gui3. when opening game rename the 2 gui2 to eachother so origninal one is named the default. I have made some styles which are in official WoT discord under Community Creations (Vurdu_Ve_Gol_Oldu) If still need help i can help you indivicualy, just dm me on discord (wellinthatcasethatsucks)

**Comment by shanstac** [comment]
After doing what both Cerberus and QuietShow said, I am now getting this error message when trying to run TankExporter 
![4edca69cb7b9b4f1cea286dc2660af71](https://github.com/mikeoverbay/TankExporter/assets/61248325/10937ce8-eff9-445c-a9d1-d75e31cf9d98)
WoT Version is 1.25.0.4580. I am on the NA server of WoT.

**Comment by Cerberus50** [comment]
probably an error while copying and pasting from part 3 to part 2. You must
proceed with winrar

**Comment by shanstac** [comment]
What I done was I extracted the contents of both gui-part2 and gui-part3 into a completely separate folder then combined the contents into one folder archived (zipped) it calling it gui-part2 and then copying that back into the res/package folder overwriting the original. But I finally figured it out by just opening the gui-part2(original) while being in the res/package folder, and just adding the contents that I had extracted from gui-part3 and T.E. started working for me. 

Do have a question though, has anyone ever tried to use T.E. on the Lesta MirTankov game files?
